schema/queries.proto

```diff
@@ -21,7 +21,7 @@ message GetAccountAssetTransactions {
 }
 
 message GetTransactions {
-  repeated string tx_hashes = 1;
+  repeated bytes tx_hashes = 1;
```

Comment:
Why bytes? Quite inconsistent with the rest hash representation

Reply:
Followings are using `bytes`. In Iroha, Blob is used to express pubkey, signature, hash and they all are using `bytes`.
https://github.com/hyperledger/iroha/blob/develop/schema/endpoint.proto#L21-L28
https://github.com/hyperledger/iroha/blob/develop/schema/responses.proto#L83